PersonsApi/appsettings.json

```diff
@@ -0,0 +1,43 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
```

Comment:
Please set a sensible default log level to 'warnings' or greater to avoid excessive log generation